manifests/init.pp

```diff
@@ -102,6 +102,9 @@
   }
 
   $nss_db_dir   = "${pki_dir}/nssdb"
+  $nss_db_password_file = 'nss_db_password-file'
```

Comment:
I believe you do not need this anymore. You could keep it around for compatibility, but I am OK with merging this PR and https://github.com/theforeman/puppet-katello/pull/419 together given all the changes we have going in across the modules.